docs/developer-guide.md

```diff
@@ -183,37 +183,151 @@ TestUtils.mock_postal_code()
 
 ### End-to-end Tests with Playwright
 
-#### Run Playwright Specs
+Certainly! Below is a simple template for documentation on writing Playwright tests, including best practices and tips. Feel free to customize it based on your specific needs:
 
-1.0 Ensure the server is running:
+---
 
-Start server from new terminal command:
+# Playwright Testing Documentation
+
+## Introduction
+
+Playwright is a powerful end-to-end testing library for web applications. It provides a simple API to automate browser actions, allowing developers to create robust and maintainable tests.
+
+## Getting Started
+
+To get started with Playwright, you'll need to install it as a dependency in your project:
 
 ```bash
-cd bc_obps && make run
+npm install @playwright/test
```

Comment:
Should this be `yarn add @playwright/test` or even just a reminder to run `cd client && yarn`? 

I think if someone uses `nom install` in this directory they will risk committing a `package-lock.json`.